app/helpers/select_options_helper.rb

```diff
@@ -23,6 +23,16 @@ def select_provider_options(providers)
     ] + providers.map { |provider| OpenStruct.new(id: provider.id, name: "#{provider.name} (#{provider.code})") }
   end
 
+  def select_sort_options
+    [
+      [0, 'All time'],
```

Comment:
Could the values here be strings, e.g. `all_time` to make the URL params more transparent?

Reply:
Yep good idea